### PR TITLE
Convert to Python3

### PR DESCRIPTION
**Pull request by ralic**

### CODE REVIEW

File: CommonMark/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # flake8: noqa
-from __future__ import unicode_literals, absolute_import
```
Comment (nikolas) [inline CommonMark/__init__.py]: these import are used to maintain compatibility with python 2, which this library still supports, and will for a while: https://github.com/rtfd/CommonMark-py/blob/master/.travis.yml#L4

**Comment by nikolas** [comment]
This library already supports python 3. Is there something specific that wasn't working for you?

File: CommonMark/common.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+
 
 import re
 import sys
 
 try:
     from urllib.parse import quote
 except ImportError:
-    from urllib import quote
+    from urllib.parse import quote
```
Comment (nikolas) [inline CommonMark/common.py]: this change doesn't make sense - the whole point is to import `quote` from `urllib` if `url.parse` fails, i.e., we are on python 2.

Comment (ralic) [inline CommonMark/common.py]: You are right.. Sorry for the infant updates.

**Pull request closed without merging**